vllm/utils.py

```diff
@@ -232,8 +232,12 @@ def get_open_port() -> int:
             return s.getsockname()[1]
 
 
-def set_cuda_visible_devices(device_ids: List[int]) -> None:
-    os.environ["CUDA_VISIBLE_DEVICES"] = ",".join(map(str, device_ids))
+def update_environment_variables(envs: Dict[str, str]):
+    for k, v in envs.items():
+        if k in os.environ:
+            logger.warning(f"Overwriting environment variable {k} "
+                           f"from {os.environ[k]} to {v}")
```

Comment:
nit: do `{os.environ[k]=}` and `{v=}` so whitespace is obvious

Reply:
I add single quotes to make it clear. `{os.environ[k]=}` will print `os.environ[k]` to users, which is confusing and not informative.